src/Agent.Plugins/PipelineCache/PipelineCacheTaskPluginBase.cs

```diff
@@ -29,6 +29,7 @@ public abstract class PipelineCacheTaskPluginBase : IAgentTaskPlugin
         protected const string ContentFormatVariableName = "AZP_CACHING_CONTENT_FORMAT";
         public Guid Id => PipelineCachePluginConstants.CacheTaskId;
         public abstract String Stage { get; }
+        public const string ResolvedFingerPrintVariableName = "RESTORE_STEP_RESOLVED_FINGERPRINT";
```

Comment:
>const [](start = 15, length = 5)

[https://exceptionnotfound.net/const-vs-static-vs-readonly-in-c-sharp-applications/](https://exceptionnotfound.net/const-vs-static-vs-readonly-in-c-sharp-applications/) 
Just for your reference - especially because there's dependencies between task and agent etc.